src/main/kotlin/no/nav/k9punsj/integrasjoner/dokarkiv/SafDtos.kt

```diff
@@ -176,6 +179,12 @@ internal object SafDtos {
             false -> false
         }
         val ikkeErTemaOMS = tema?.let { Tema.OMS.name != it } ?: false
+        private val ferdigstilteStatuser = listOf("JOURNALFOERT", "FERDIGSTILT")
+
+        val erFerdigstilt = ferdigstilteStatuser.contains(journalstatus)
+        private val erInngående = journalposttype == "I"
+        private val erNotat = journalposttype == "N"
+        internal val kanKopieres = erInngående || erNotat
```

Comment:
Eg ville ha definert enums for JOURNALFOERT, FERDIGSTILT, I og N

Reply:
Enig 👍